blog.html

```diff
@@ -0,0 +1,63 @@
+      </div>
+
+      <div class="bloggo-card">
+        <img class="bloggoimg" src="images/blog-2.jpg" alt="">
+        <h4>How to Walk Multiple Dogs</h4>
+        <p>You are doin me a concern long water shoob heck pupper fluffer maximum borkdrive long woofer pats, clouds heck extremely cuuuuuute shooberino smol. I am bekom fat super chub. The neighborhood pupper big ol pupper waggy wags bork ruff, wow very biscit yapper such treat heckin angery woofer, bork stop it fren ruff. Extremely cuuuuuute heckin you are doin me a concern, he made many woofs. Very hand that feed shibe most angery pupper I have ever seen heckin good boys and girls shibe, lotsa pats. Floofs doge lotsa pats very taste wow heck you are doin me a concern, heckin good boys and girls mlem wrinkler you are doin me a concern. Big ol pupper puggo maximum borkdrive puggorino thicc long water shoob, blop corgo blep shoob. Heckin good boys what a nice floof heck you are doin me a concern snoot what a nice floof many pats snoot heckin heck, heckin good boys tungg h*ck long doggo adorable doggo you are doin me a concern bork.</p>
+      </div>
+
+      <div class="bloggo-card">
+        <img class="bloggoimg" src="images/blog-3.jpg" alt="">
+        <h4>Teach Your Dog to Fetch</h4>
+        <p>You are doin me a concern long water shoob heck pupper fluffer maximum borkdrive long woofer pats, clouds heck extremely cuuuuuute shooberino smol. I am bekom fat super chub. The neighborhood pupper big ol pupper waggy wags bork ruff, wow very biscit yapper such treat heckin angery woofer, bork stop it fren ruff. Extremely cuuuuuute heckin you are doin me a concern, he made many woofs. Very hand that feed shibe most angery pupper I have ever seen heckin good boys and girls shibe, lotsa pats. Floofs doge lotsa pats very taste wow heck you are doin me a concern, heckin good boys and girls mlem wrinkler you are doin me a concern. Big ol pupper puggo maximum borkdrive puggorino thicc long water shoob, blop corgo blep shoob. Heckin good boys what a nice floof heck you are doin me a concern snoot what a nice floof many pats snoot heckin heck, heckin good boys tungg h*ck long doggo adorable doggo you are doin me a concern bork.</p>
+      </div>
+
+    </div>
+
+    <footer>
+      <img src="images/adoptadog-logo.svg" style="width:500px" alt="Adopt-a-dog logo.">
```

Comment:
You'll want to get rid of all your inline css and do it all from your css files

Reply:
Done.